File: launchdarkly/typescript-axios/src/DeleteRepositoryExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.CodeReferencesApi(configuration).deleteRepository(
  "repo_string", // repo
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling CodeReferencesApi#deleteRepository:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});

File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteRepositoryExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteRepositoryExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new CodeReferencesApi(config).DeleteRepository(
                repo: "repo_string"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling CodeReferencesApi#DeleteRepository: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
